tests/performance/load-tests/load-test.sh

```diff
@@ -107,8 +110,10 @@ function runTest() {
       succeeded=$((succeeded + 1))
     else
       print_error "Timeout waiting for dw$i to become ready or an error occurred."
+      ws_name=$(kubectl get dw dw$i --template='{{.status.devworkspaceId}}')
```

Comment:
If `devworkspaceId` != `workspace name`, then I would recommend to name variable correspondingly as `devworkspace_id` to avoid confusion.